sdk/core/core-tracing/src/useInstrumenter.browser.ts

```diff
@@ -0,0 +1,16 @@
+import { NoOpInstrumenter } from "./instrumenter";
+import { Instrumenter } from "./interfaces";
+
+/** @internal */
+globalThis.instrumenterImplementation = new NoOpInstrumenter();
```

Comment:
In the browser we set it on the global object to have a single instrumenter to set.

In Node, the instrumentation package will patch any `require` of core-tracing and set the instrumenter at the module level.

Let me know if this is not the way to go...

Reply:
it feels weird to expose the noop instrumenter if none is set yet... I wonder if we could do something for the browser like "if a global instrumenter is set, use that, otherwise us the one we've cached with useInstrumenter" - that way the global variable functions more like an environment variable override